shared/checks/oval/install_mcafee_hbss.xml

```diff
@@ -4,7 +4,16 @@
     <metadata>
       <title>Install McAfee Host-Based Intrusion Detection Software (HBSS)</title>
       <affected family="unix">
-        <platform>multi_platform_all</platform>
+	<platform>multi_platform_debian</platform>
+	<platform>multi_platform_fedora</platform>
+	<platform>multi_platform_opensuse</platform>
+	<platform>multi_platform_ol</platform>
+	<platform>multi_platform_rhcos</platform>
+	<platform>multi_platform_rhel</platform>
+	<platform>multi_platform_rhv</platform>
+	<platform>multi_platform_sle</platform>
+	<platform>multi_platform_ubuntu</platform>
+	<platform>multi_platform_wrlinux</platform>
```

Comment:
@matejak Could you clarify why this is needed?
Why `multi_platform_all` doesn't work?

Reply:
Those are Linux OS products. You don't want this OVAL to appear in products s.a. OSP, Firefox and whatnot. I think that a shared OVAL makes it into a benchmark even if there is no rule for it, and then it can happen that it is "incomplete". As no makes use of it, it's not a real issue, but as we don't distinguish between useful and useless OVALs, a missing extending definition is treated in the same way for all OVALs.

Reply:
Thanks for the clarification, and I think your observations are correct.